files/jediluke.space.txt

```diff
@@ -0,0 +1,17 @@
+server {
```

Comment:
This file looks good, but it has kind of a weird name - shouldn't be something like `jediluke_space_nginx_conf.txt` ?